source/images/gem.svg

```diff
@@ -1,21 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<svg width="41px" height="36px" viewBox="0 0 41 36" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
-    <!-- Generator: Sketch 3.0.4 (8054) - http://www.bohemiancoding.com/sketch -->
-    <title>gem</title>
-    <desc>Created with Sketch.</desc>
-    <defs></defs>
-    <g id="Page-2" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
-        <g id="Flag-3" sketch:type="MSArtboardGroup" transform="translate(-74.500000, -16.244444)" fill="#CCE3FF">
-            <g id="gem" sketch:type="MSLayerGroup" transform="translate(77.000000, 19.000000)">
-                <path d="M16.0198172,0 L7.29749895,0 L10.8786249,7.1939916 L16.0198172,0" id="Fill-1" sketch:type="MSShapeGroup"></path>
-                <path d="M22.7576352,9.1366777 L17.9911788,1.70056414 L12.6763285,9.1366777 L22.7576352,9.1366777" id="Fill-2" sketch:type="MSShapeGroup"></path>
-                <path d="M35.6437478,9.1366777 L29.6588289,1.65088345 L26.0788454,9.1366777 L35.6437478,9.1366777" id="Fill-3" sketch:type="MSShapeGroup"></path>
-                <path d="M27.6983209,0 L19.9431181,0 L24.3839657,6.92902795 L27.6983209,0" id="Fill-4" sketch:type="MSShapeGroup"></path>
-                <path d="M5.46380706,1.91013668 L-0.284045837,9.1366777 L9.06207036,9.1366777 L5.46380706,1.91013668" id="Fill-5" sketch:type="MSShapeGroup"></path>
-                <path d="M0.270631679,10.8498048 L15.3143313,28.3625318 L9.8949435,10.8498048 L0.270631679,10.8498048" id="Fill-6" sketch:type="MSShapeGroup"></path>
-                <path d="M20.590337,28.4150676 L35.0787879,10.8498048 L25.4561898,10.8498048 L20.590337,28.4150676" id="Fill-7" sketch:type="MSShapeGroup"></path>
-                <path d="M17.9911788,29.7039102 L23.21463,10.8498048 L12.1553543,10.8498048 L17.9911788,29.7039102 Z" id="Fill-8" sketch:type="MSShapeGroup"></path>
-            </g>
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<svg width="41" height="36" viewBox="0 0 41 36" xmlns="http://www.w3.org/2000/svg"><title>gem</title><desc>Created with Sketch.</desc><g fill="none" fill-rule="evenodd"><g fill="#CCE3FF"><g><path d="M18.52 2.756H9.797L13.38 9.95l5.14-7.194"/><path d="M25.258 11.893L20.49 4.457l-5.314 7.436h10.082"/><path d="M38.144 11.893L32.16 4.407l-3.58 7.486h9.564"/><path d="M30.198 2.756h-7.755l4.44 6.93 3.315-6.93"/><path d="M7.964 4.666l-5.748 7.227h9.346L7.964 4.666"/><path d="M2.77 13.606L17.815 31.12l-5.42-17.514H2.772"/><path d="M23.09 31.17l14.49-17.564h-9.624L23.09 31.17"/><path d="M20.49 32.46l5.225-18.854h-11.06L20.49 32.46z"/></g></g></g></svg>
```

Comment:
I've been using `svgo` to clean up the `svg`s, thanks to Steven's suggestion

I'll remove the desc